ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -16,18 +27,15 @@ <h4 class="header-title m-t-0 m-b-20">Speaking Practice</h4>
                     <h1 class="card-title">Review complete!</h1>
 					<!--table for displaying results of a practice-->
                     <table id="results_table" class="table table-hover"> 
-                        <thead>
-                            <tr>
-                                <th>Student Transcription</th> 
-                                <th class="text-center">Accuracy Score</th>
-                                <th class="text-center">Fluency Score</th> 
-								<th class="text-center invisible"><button class="show-more" type="button">Show More</button></th>
-							</tr>
-                        </thead>
+						<div class="row header">
```

Comment:
There is some funky stuff going on because this file uses both tabs and spaces to control indentation. It should be made consistent using just spaces. In VS Code, you can press Ctrl+Shift+P and then type "Convert Indentation to Spaces" to do this easily.